src/coreclr/src/inc/corinfo.h

```diff
@@ -1176,6 +1317,7 @@ struct CORINFO_METHOD_INFO
     CorInfoRegionKind           regionKind;
     CORINFO_SIG_INFO            args;
     CORINFO_SIG_INFO            locals;
+    CORINFO_OSR_INFO            osrInfo;
```

Comment:
Would it be more appropriate for this to be returned by a new JIT/EE interface method that the JIT calls when it gets the `CORJIT_FLAG_OSR` flag?

Reply:
Probably, yes. I think I can cache it on the runtime interface side.

Reply:
Jit now pulls OSR info instead of runtime pushing it.